client/src/assets/css/ChanStrip.css

```diff
@@ -168,7 +168,7 @@
     outline: none;
     -moz-outline: none;
     color: white;
-    height: 62px;
+    height: 55px;

```

Comment:
Out of curiosity: What is the motivation behind the pixel change?

Reply:
@LindvedKrvang the commit where it's added is: `[add Mono input selector make buttons smaller 2 fit 4 options]`
So basically before there were 3 options now there are 4.
A better approach would arguably be to let it be dynamic, but this way, as little code as possible was changed.
🙂